packages/psss/src/store/store.js

```diff
@@ -22,6 +21,4 @@ const enhancers = composeEnhancers(applyMiddleware(thunk));
 
 const store = createStore(persistedReducer, {}, enhancers);
 
-apiErrorHandler(store);
```

Comment:
Removing the global 401 handler for now as it introduced some regressions around catching 401s that should be caught elsewhere in the app. I have created another card for it so that it can be done carefully later and tested properly. @see https://app.zenhub.com/workspaces/sprint-board-5eea9d3de8519e0019186490/issues/beyondessential/tupaia-backlog/1748